src/hooks/useProcessDetails.tsx

```diff
@@ -220,6 +222,13 @@ export function useProcessDetails(processId: number): IProcessDetails {
   ): Promise<void> => {
     try {
       let newProcess = await updateProcessStage(newStage, assignee);
+      //updateProcessStage successful, save metrics event
+      const matches = noteText.match(/<strong>(.*?)<\/strong>/g);
+      await metricsApi.submitEvent(
+        newProcess.CurrentStage,
+        newProcess.Id,
+        matches?.[0].replace(/<\/?strong>/g, "")
+      );
```

Comment:
We need to save the metric event, but the Reason should just be undefined -- as we should never have a rework reason moving forward. Will prevent it from just grabbing whatever text they may bold in the note of an event and putting it in this reason field.